README.md

```diff
@@ -208,6 +230,16 @@ wiki page.
    you a hint what is wrong with your `.asoundrc` entry. Also, take a look at the
    [bluealsa-plugins man page](./doc/bluealsa-plugins.7.rst).
 
+## Contributing
+
+> policy for accepting PRs, coding and design standards, anything to encourage useful code/doc contributions.
+
+> advice for raising issues.
```

Comment:
Maybe PR/issue template might be a good idea? But in the readme the common sense advice might be included such as "please, search for similar issues before raising a new one" :D

Reply:
I'll try to think up some more "common sense" advice and put it in here. PR/issue templates are really up to your own preference. They have pros and cons, I have no particular preference either way.

Reply:
hmm, just tried mermaid again, and now it works. Here's an attempt to reproduce my context diagram:
```
flowchart TD
    classDef external fill:#eee,stroke:#333,stroke-width:4px;
    classDef bluealsa fill:#bbf,stroke:#333,stroke-width:4px;

    A[Bluetooth Adapter] <--> B((bluetoothd daemon))
    A <---> C((bluealsa daemon))
    B <--> C
    C <--> D((bluealsa-aplay))
    C <--> E(("ALSA clients
(via plugin)"))
    C <--> F((D-Bus clients))

    class A,B external;
    class C,D,E,F bluealsa;

```
```mermaid
flowchart TD
    classDef external fill:#eee,stroke:#333,stroke-width:4px;
    classDef bluealsa fill:#bbf,stroke:#333,stroke-width:4px;

    A[Bluetooth Adapter] <--> B((bluetoothd daemon))
    A <---> C((bluealsa daemon))
    B <--> C
    C <--> D((bluealsa-aplay))
    C <--> E(("ALSA clients
(via plugin)"))
    C <--> F((D-Bus clients))

    class A,B external;
    class C,D,E,F bluealsa;

```

Reply:
OK, that's great :)
However it is unreadable when using GitHub dark mode. I think that you should also set color for text. Otherwise its bright text on bright background.

> Diagrams in READMEs are rare. The fact is they can be hard to maintain, and add bulk to the source code download.

It does not have to be in the README. It can be in wiki article, and README can point to article e.g. "BlueALSA Design"